dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-jdbc/src/main/java/org/apache/dolphinscheduler/plugin/registry/jdbc/JdbcRegistryConfiguration.java

```diff
@@ -34,7 +34,7 @@
 @ConditionalOnProperty(prefix = "registry", name = "type", havingValue = "jdbc")
 public class JdbcRegistryConfiguration {
 
-    @Bean
+    @Bean("sqlSessionFactory")
```

Comment:
It's not a good idea add name 'sqlSessionFactory' for this bean, since this will cause the JdbcRegistry use the same sql session factory with the business.

Reply:
> It's not a good idea add name 'sqlSessionFactory' for this bean, since this will cause the JdbcRegistry use the same sql session factory with the business.

Thank you for reminding me of this. 
I write this because I find that if I don't explicitly declare 'sqlSessionFactory' , an exception will be thrown at startup:
[dolphinscheduler-standalone.log](https://github.com/apache/dolphinscheduler/files/13303243/dolphinscheduler-standalone.log)


I think it might be because of 'daoconfiguration. Java 'references it
<img width="972" alt="image" src="https://github.com/apache/dolphinscheduler/assets/40498606/803b41ea-d4c7-4cf1-b72a-0e52243e2588">


Is this a known problem?

Reply:
> > It's not a good idea add name 'sqlSessionFactory' for this bean, since this will cause the JdbcRegistry use the same sql session factory with the business.
> 
> Thank you for reminding me of this. I write this because I find that When I  using `JDBCRegistry plugin`,
 if I don't explicitly declare 'sqlSessionFactory' , an exception will be thrown at startup: [dolphinscheduler-standalone.log](https://github.com/apache/dolphinscheduler/files/13303243/dolphinscheduler-standalone.log) <img alt="image" width="1106" src="https://user-images.githubusercontent.com/40498606/281602379-e26c6a96-4dfa-415e-ba99-2c1b618850a9.png">
> 
> I think it might be because of 'DaoConfiguration. java 'references it <img alt="image" width="972" src="https://user-images.githubusercontent.com/40498606/281602082-803b41ea-d4c7-4cf1-b72a-0e52243e2588.png">
> 
> Is this a known problem?

Reply:
@ruanwenjun  can you help me? What can I do to improve this feature?

Reply:
I didn't meet this problem, it can work well when I use JDBC registry